lib/ui/LandingPage.dart

```diff
@@ -0,0 +1,177 @@
+  final LandingPageViewModel _viewModel;
+  final LandingPageStyle _style;
+
+  const LandingPage(
+      {Key key,
+      LandingPageViewModel viewModel,
+      LandingPageStyle style = _defaultStyle,})
+      : this._viewModel = viewModel,
+        this._style = style,
+        super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      margin: _Constants.generalPadding,
+      child: Column(
+        children: <Widget>[
+          Flexible(
+            flex: _Constants.imageFlexValue,
+            fit: FlexFit.loose,
+            child: Container(
+              child: Image.asset(
+                _viewModel.headerImage,
+                width: double.infinity,
+              ),
+            ),
+          ),
+          Container(
+            height: _Constants.bottomContainerHeight,
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.spaceAround,
+              children: <Widget>[
+                Column(
+                  mainAxisAlignment: MainAxisAlignment.end,
+                  children: <Widget>[
+                    Image.asset(_viewModel.subtitleImage),
+                    SizedBox(height: _Constants.subtitleLineSpace,),
+                    Text(
+                      Intl.message(_viewModel.detailText),
+                      style: _style.detailTextStyle,
+                      maxLines: _style.detailTextMaxLines,
+                      overflow: TextOverflow.ellipsis,
+                      textAlign: TextAlign.center,
+                    ),
+                  ],
+                ),
+                Column(
+                  mainAxisAlignment: MainAxisAlignment.end,
+                  children: <Widget>[
+                    RoundedButton(
+                      viewModel: RoundedButtonViewModel(
+                          title: Intl.message(_viewModel.actionText),
+                          onTap: () {},),
+                      style: RoundedButtonStyle.largeRoundedButtonStyle()
+                          .copyWith(buttonTextStyle: _style.actionTextStyle),
+                    ),
+                    SizedBox(height: _Constants.actionLineSpace,),
+                    GestureDetector(
+                      onTap: () => _onMenuPressed(context),
+                      child: Text(
+                        Intl.message(_viewModel.footerText),
+                        style: _style.footerTextStyle,
+                        maxLines: _style.footerTextMaxLines,
+                        overflow: TextOverflow.ellipsis,
+                      ),
+                    ),
+                  ],
+                ),
+              ],
+            ),
+          )
+        ],
+      ),
+    );
+  }
+
+  Future _onMenuPressed(BuildContext context) async {
+    showModalBottomSheet(
+        context: context,
+        builder: (widgetBuilder) => ActionSheet(
+            viewModel: MockActionSheetViewModel.buildWithCheckBox(),
```

Comment:
You need to implement your own `ActionSheetViewModel` 
Don't use the `MockActionSheetViewModel`

Reply:
Done ! Thanks for the explanation !

Reply:
👍